amd/src/iconpicker.js

```diff
@@ -1,15 +1,23 @@
 import {exception as displayException} from 'core/notification';
 import Templates from 'core/templates';
+import Modal from 'core/modal';
 import ICON_SET from 'block_floatingbutton/iconset';
 
+const SELECTORS = {
+  iconpicker: ".block_floatingbutton-iconpicker",
+  iconpicker_input: ".block_floatingbutton-input input",
+  iconpicker_search_input: ".block_floatingbutton-iconpicker-search-input",
+  iconpicker_icon: ".block_floatingbutton-iconpicker-icon",
```

Comment:
grunt is complaining that these have to be camel case (or, as constants, uppercase would also be possible I believe)